src/components/DiagnosticsBody.vue

```diff
@@ -62,14 +97,32 @@ export default Vue.extend({
     <sortable-table
       key-field="id"
       :headers="headers"
-      :rows="rows"
+      :rows="filteredRows"
       :search="false"
       :table-actions="false"
       :row-actions="false"
       :sub-rows="true"
       :sub-expandable="true"
       :sub-expand-column="true"
     >
+      <template #no-rows>
+        <td :colspan="headers.length + 1">
+          <empty-state
+            icon="icon-search"
+            heading="No results found"
+            body="Try showing muted diagnostics to see some results."
```

Comment:
This will be incorrect in the future when there are no failing diagnostics. I rewrote it this way:
```
:body="emptyStateSuggestion"
```

and in `computed:`:
```

    emptyStateSuggestion(): string {
      if (this.hideMuted && this.rows.length > 0) {
        return 'Try showing muted diagnostics to see some results.';
      } else {
        return 'Everything appears to be functioning correctly!';
      }
    },
```